### DISCUSSION TITLE
support ARM

### DISCUSSION THREAD

**Discussion by zhangxinmrs** [azerothcore/azerothcore-wotlk#3928] [Ideas]
Anyone can help to merge this ARM patch, thanks!
https://github.com/zhangxinmrs/Azerothcore-wotlk-ARM

**Comment by FrancescoBorzi** [comment]
@zhangxinmrs http://www.azerothcore.org/wiki/How-to-create-a-PR

**Comment by Helias** [comment]
@zhangxinmrs make a pull request.

**Comment by BarbzYHOOL** [comment]
He meant this link: http://www.azerothcore.org/wiki/How-to-create-a-PR (create, not test)

**Comment by FrancescoBorzi** [comment]
oh yes, my bad, thanks @BarbzYHOOL 

@zhangxinmrs let us know if you need any help!
